bundles/org.openhab.binding.cp750/src/main/resources/OH-INF/i18n/cp750.properties

```diff
@@ -0,0 +1,3 @@
+# FIXME: please add all English translations to this file so the texts can be translated using Crowdin
```

Comment:
After the thing-types.xml is corrected, the i18n plugin needs to be run to add the default translations.